.travis.yml

```diff
@@ -2,37 +2,54 @@ language: node_js
 
 jobs:
   fast_finish: true
+
   include:
     - stage: Lint
       env: SCRIPT=lint
       node_js: 'stable'
+
+    - &osx
+      stage: Test
+      os: 'osx'
+      env: SCRIPT=test
+      node_js: 'stable'
+    - <<: *osx
+      node_js: 'lts/*'
```

Comment:
Better use `8`, i think we should use `lts/NAME_OF_RELEASE` or number everywhere than mix they. Also think we should add `node@10` and latest night build to avoid regressions and broken package (for `linux` and `windows`)